service/query/service.go

```diff
@@ -80,9 +86,44 @@ func (svc *Service) ListValidators(_ client.ListValidatorsRequest, reply *client
 		return codes.ErrListValidators
 	}
 
+	stakingOptions, err := svc.govern.GetStakingOptions()
+	if err != nil {
+		svc.logger.Error("error to fetch staking options")
+		return codes.ErrListValidators
+	}
+
+	vMap := make(map[string]bool)
+	cnt := int64(0)
+
+	i := 0
+	queue := &identity.ValidatorQueue{PriorityQueue: make(utils.PriorityQueue, 0, 100)}
```

Comment:
Why do you need to use the queue to store the validators? Is there a reason you can't set them directly to vMap?